pandas/tests/test_resample.py

```diff
@@ -2960,7 +2960,7 @@ def test_methods(self):
             expected = g.B.apply(lambda x: getattr(x.resample('2s'), f)())
             assert_series_equal(result, expected)
 
-        for f in ['backfill', 'ffill', 'asfreq']:
+        for f in ['nearest', 'backfill', 'ffill', 'asfreq']:
```

Comment:
let's add a test for this specifically to make sure its doing the right thing. IOW construct a frame with an expected result (rather than the auto generated one here which might or might now actually do nearest resampling, as it doesn't have nans).

Reply:
something like the following will work? 
```python 
   def test_nearest(self):
        index = pd.date_range('1/1/2000', periods=3, freq='T')
        result = pd.Series(range(3), index=index).resample('20s').nearest()

        expected= pd.Series([0, 0, 1, 1, 1, 2, 2], index= pd.DatetimeIndex(['2000-01-01 00:00:00', '2000-01-01 00:00:20',
               '2000-01-01 00:00:40', '2000-01-01 00:01:00',
               '2000-01-01 00:01:20', '2000-01-01 00:01:40',
               '2000-01-01 00:02:00'],
              dtype='datetime64[ns]', freq='20S'))
        assert_series_equal(result,  expected)
```